nestkernel/completed_checker.cpp

```diff
@@ -64,6 +64,34 @@ CompletedChecker::all_true() const
   return true;
 }
 
+bool
+CompletedChecker::any_false() const
```

Comment:
why not use `std::any_of` since we use c++11 now? we should probably also replace the other functions, maybe in a separate PR though